tests/rtl/mockServices/userMockServices.js

```diff
@@ -0,0 +1,21 @@
+import { rest } from 'msw';
+import faker from 'faker';
+
+import userFactory from 'fixtures/userFactory';
+
+const mockLoginRequest = rest.post(/\/users\/sign_in/i, async (req, res, ctx) => {
```

Comment:
```suggestion
const mockLoginRequest = rest.post('/users/sign_in', async (req, res, ctx) => {
```

Be careful here, this regex compare if contains this paths, but if you call in real services to `/foo/users/sign_in/bar` it's going to intercept that route this test its going to work anyway, in the future this will pass many tests that call another routes. Is more secure to compare just with a little bit strict string.

For example if you have another different post service `/accounts/users/sign_in` `msw` it going to intercept this one.

Also it will be a good idea when no route is matching with for example at end of this handlers, match res.post(*, (req, res, ctx) => { response and throw and exception here }) in order to know whats going on when route is not matching

Reply:
I have some problems with Circle CI because the origin of the request is random (i.e 'circle-repo/users/sign_in'), I have to figure out how to solve that problem in an efficient way

Reply:
maybe enforce some env variable for the origin? it just an idea

Reply:
Fixed, it was the circle CI variable for the API URL. For change that I deleted from circle CI the variable and change a little bit `config.yml` to set `.env.test` in cypress running